packages/api-demo/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/index.js",
   "scripts": {
     "serve:no-browser": "npm run serve -- --no-browser",
-    "serve": "live-server --mount=/:src --mount=/scripts/containerjs-api-bundle:node_modules/containerjs-api-bundle/build/containerjs-bundle.js",
+    "serve": "live-server --mount=/:src --mount=/scripts/containerjs-api-bundle:node_modules/containerjs-api-bundle/build/containerjs-bundle.js --mount=/notifications/notification.html:node_modules/containerjs-api-bundle/build/notification.html",
```

Comment:
Think we should be able to do `--mount=/scripts/containerjs-api-bundle:node_modules/containerjs-api-bundle/build/` to serve both files together